ompi/mca/mtl/psm2/help-mtl-psm2.txt

```diff
@@ -47,5 +47,19 @@ Unknown path record query mechanism %s. Supported mechanisms are %s.
 Message size %llu bigger than supported by PSM2 API. Max = %llu
 #
 [no psm2 cuda env]
-Using CUDA enabled OpenMPI but PSM2_CUDA environment variable is %s.
-This is not a recommended combination. If the application uses %s.
+Warning: Running an Open MPI build with CUDA support over Intel(r) Omni-Path
+without enabling CUDA support on PSM2 library. Open MPI has therefore defaulted
+to setting PSM2_CUDA=1. This may impact performance if NOT running CUDA aware
+applications. Set your environment with variable PSM2_CUDA equal to 1 to clear
+this message, or set it to 0 to hint PSM2 that no CUDA support is needed.
```

Comment:
Minor tweak suggestions for this message (per below, I'm assuming you'll only emit this message when there are CUDA devices present):

> Warning: Open MPI has detected that you are running in an environment with CUDA devices present and that you are using Intel(r) Ompi-Path networking.  However, the environment variable PSM2_CUDA was not set, meaning that the PSM2 Omni-Path networking library was not told how to handle CUDA support.
> 
> If your application uses CUDA buffers, you should set the environment variable PSM2_CUDA to 1; otherwise, set it to 0.  Setting the variable to the wrong value can have performance implications on your application, or even cause it to crash.
>
> Since it was not set, Open MPI has defaulted to setting the PSM2_CUDA environment variable to 1.

Reply:
Ok, Will fix this